sdk/cmake/msvc.cmake

```diff
@@ -85,6 +85,12 @@ add_compile_flags("/wd4018")
 # - C4603: macro is not defined or definition is different after precompiled header use
 # - C4716: function must return a value
 add_compile_flags("/we4013 /we4020 /we4022 /we4047 /we4098 /we4113 /we4129 /we4163 /we4229 /we4700 /we4603 /we4716")
+if((NOT ARCH STREQUAL "amd64") AND (NOT ARCH STREQUAL "arm"))
```

Comment:
Switched to `if(ARCH STREQAL "i386")`.
I didn't know such ports were currently supported.
My idea was more to enable it by default, except where "known" to be "broken". Otherwise, it seems likely to be forgotten. Anyway...